### DISCUSSION TITLE
Open relevant business rules from slash command

### DISCUSSION THREAD

**Discussion by cnavr** [arnoudkooi/SN-Utils#271] [Ideas]
It would be useful to have something like "-b" parameter in slashcommand and open all relevant BRs for current table.

**Comment by VanWeapon** [comment]
You can do this in a slashcommand like `/rules` and make the url `/sys_script_list.do?sysparm_query=collection=$table`

![image](https://user-images.githubusercontent.com/30616047/180599597-54e08205-1f96-42e7-ac5c-b003fc892541.png)

**Reply by cnavr** [reply]
This would help only for table-specific BRs, but I will not see BRs from the parent tables (if the table is extended).
Any idea, how can I see also BRs from parent tables without using "/-c" and looking at BRs after?
Thanks
Ctirad

**Comment by VanWeapon** [comment]
Otherwise you can use the `-c` flag after typing the table name to go to the universal config page for that table, which includes business rules and all other table config

**Comment by cnavr** [comment]
Hi Luke,
But I meant similar functionality like if u use "/-t" and open table
definition of current table or "/-c"  and open all configurations for
current table.
It would be nice to have "/-b" to open directly BRs instead of using "/-c"
and go to BRs.
Ctirad

**Comment by arnoudkooi** [comment]
For now I would recommend creating a custom slashcommand. I may look at customizing "switches" in the future

**Reply by cnavr** [reply]
OK, thanks.
Ctirad

**Comment by vedang444** [comment]
We do have the ability to build our own switches now. So that could work. Here is the one I use for Business Rules:
```
"br": {
        "description": "Business Rules on table",
        "value": "sys_script_list.do?sysparm_query=collection=$0",
        "type": "link"
    },
```

Arnoud has put some infomration on it here:
[https://www.arnoudkooi.com/switch/](https://www.arnoudkooi.com/switch/)
